frontend/src/store/modules/auth/types/actions.js

```diff
@@ -1,4 +1,5 @@
 export const LOGIN = 'login';
+export const UPDATE = 'update';
```

Comment:
rename this action to something more specific like `UPDATE_USER` or `UPDATE_AUTHORIZED_USER`